File: src/codes/Users_Manual/controller_1.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Gain;
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Utils;
~using var autd = new ControllerBuilder().AddDevice(new AUTD3(Vector3d.Zero)).Open(Nop.Builder());
~var m = new Static();
~var g = new Null();
autd.Send((m, g), TimeSpan.FromMilliseconds(20));

File: src/codes/Users_Manual/controller_1.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~auto autd =
//~autd3::ControllerBuilder({autd3::AUTD3(autd3::Vector3::Zero())}).open(autd3::link::Nop::builder());
//~autd3::Static m;
//~autd3::Null g;
autd.send((m, g).with_timeout(std::chrono::milliseconds(20)));
//~return 0; }